soccer/gameplay/tactics/defense.py

```diff
@@ -71,6 +113,21 @@ def execute_running(self):
             # raise RuntimeError("Defense tactic requires a goalie id to be set")
 
             # TODO: move a lot of this code into modules in the evaluation folder
+
+            #main.system_state().draw_circle(robocup.Point(0, 0), constants.Field.ArcRadius * 2,constants.Colors.Red, "Clear Ball")
+
+    def execute_clearing(self):
```

Comment:
Take a look at the `Field_Dimensions` class - I think it's got a goal zone shape with a "contains point" method you can use here.